system/helpers/numberUtils.cfm

```diff
@@ -0,0 +1,29 @@
+<cffunction name="numberThousandDecimalFormat" access="public" returntype="string" output="false">
```

Comment:
The thousand is misleading in the name here - could be any precision. I think name this `presideStandardNumberFormat()` - we can then make it do more later if wanted.

Reply:
Rename helper name.